itests/common.bash

```diff
@@ -275,6 +281,9 @@ teardown() {
 do_test() {
   _clean
   _startup
+  if [ "$1" = "poller_wait" ]; then
+    _wait_for_poller_start
+  fi
```

Comment:
there's a race condition where the sarama client (upon the first write to kafka) and the data-poller try to create the kafka topic at the same time.
Sarama only creates a topic with one partition, and bifrost throws an exception for multi-partition tests when Sarama creates the single partition topic. Because the data-poller creates the kafka topic with the correct # of partitions, I'm allowing the itest to wait some time for the data-poller to create the topic before creating `insert` statements on the DB.